backend/README.md

```diff
@@ -1,14 +1,16 @@
 # Run the backend locally using Docker

```

Comment:
@SL1015 could you document in Notion how to interact also with the deployed version?